app/api/models/statistics.py

```diff
@@ -7,9 +7,21 @@
 from loguru import logger
 
 class Homology(BaseModel):
-    coverage: float = Field(alias="homology_coverage", default=None)
-    coverage_explanation: str = None
-
+    coverage: float = Field(alias="compara.homology_coverage ", default=None)
+    reference_species_name: str = Field(alias="compara.homology_reference_species ", default=None)
```

Comment:
Does the extra space before the closing double quote matter?

Reply:
Updated code to handle both once production fixes this